lib/src/data/contributor.dart

```diff
@@ -0,0 +1,62 @@
+class Contributor {
+
+	static const List<Contributor> contributors = [
+		Contributor(
+			name: "David T.",
+			gradYear: "'23",
+			title: "Frontend",
+			email: "tarrabd@ramaz.org",
+			imageName: "https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg",
+			description: "Placeholder"
+		),
+		Contributor(
+			name: "Brayden K.",
+			gradYear: "'23",
+			title: "Backend",
+			email: "kohlerb@ramaz.org",
+			imageName: "https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg",
+			description: "Placeholder"
+		),
+		Contributor(
+			name: "Joshua T.",
+			gradYear: "'23",
+			title: "Middleend and Apple Expert",
+			email: "todesj@ramaz.org",
+			imageName: "https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg",
+			description: "Placeholder"
+		),
+		Contributor(
+			name: "Levi L.",
+			gradYear: "'21",
+			title: "Biggest Boi",
+			email: "leschesl@ramaz.org",
+			imageName: "https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg",
+			description: "Placeholder"
+		),
+		Contributor(
+			name: "Mr. Vovsha",
+			gradYear: "'21",
+			title: "Mentor",
+			email: "evovsha@ramaz.org",
+			imageName: "https://flutter.github.io/assets-for-api-docs/assets/widgets/owl.jpg",
+			description: "Placeholder"
+		)
+	];
+
+	final String name;
+	final String gradYear;
+	final String title;
+	final String email;
+	final String description;
+	final String imageName;
+
+	const Contributor({
+		required this.description,
+		required this.gradYear,
+		required this.email,
+		required this.name,
+		required this.title,
+		required this.imageName,
+	});
+	
+}
```

Comment:
```suggestion
}

```
Git complains when you don't have a new line at the end of a file, so make sure to do that